src/deps/tinycthread.c

```diff
@@ -21,6 +21,7 @@ freely, subject to the following restrictions:
     distribution.
 */
 
+#if defined(STDC_NO_THREADS)
```

Comment:
STDC_NO_THREADS has to be defined, see previous comment (may be named 'VKVG_NO_CTHREADS'.